src/threadpool.c

```diff
@@ -183,8 +184,23 @@ static void init_threads(void) {
   unsigned int i;
   const char* val;
   uv_sem_t sem;
+  char* pbuf;
+  size_t szbuf;
 
   nthreads = ARRAY_SIZE(default_threads);
+  pbuf = uv__malloc(MAX_THREADPOOL_SIZE_IN_STR * sizeof(char));
```

Comment:
Instead of dynamically allocating memory, just use a stack allocated buffer. That will also simplify the following block of code.